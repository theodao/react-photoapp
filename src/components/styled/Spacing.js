import styled from 'styled-components';

const sizeMap = {
  small: '10px',
  medium: '20px',
  large: '30px',
};

export const SpacingSizes = {
  SM: 'small',
  MD: 'medium',
  LG: 'large',
};

export default styled.div(({ size = 'md' }) => ({
  width: '100%',
  height: sizeMap[size],
}));
