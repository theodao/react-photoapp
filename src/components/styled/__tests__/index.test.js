import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Spacing from '../Spacing';

describe('Spacing style testing', () => {
  afterEach(cleanup);

  it('Should match snapshot', () => {
    const { asFragment } = render(<Spacing />);
    expect(asFragment()).toMatchSnapshot();
  });
});
