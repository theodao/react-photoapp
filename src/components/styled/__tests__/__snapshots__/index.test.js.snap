// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacing style testing Should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm jwvrSe"
  />
</DocumentFragment>
`;
